source/frontend/src/components/maps/leaflet/Control/Legend/LegendControl.tsx

```diff
@@ -16,7 +16,7 @@ import { Legend } from './Legend';
 const LegendButton = styled(Button as any)`
   &&.btn {
     background-color: #ffffff;
```

Comment:
same question here re: hardcoded color #ffffff - I'm fine if we make tackle that outside of this pr as we already have a ton of changes